eng/pipelines/merge-docs-main-to-live.yml

```diff
@@ -0,0 +1,92 @@
+parameters:
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+- name: DocsRepos
+  type: object
+  default:
+    - RepoName: azure-docs-sdk-java
+      RepoOwner: Azure
+      AllowlistPath: docsms-allowlist/java-allowlist.txt
+    - RepoName: azure-docs-sdk-python
+      RepoOwner: MicrosoftDocs
+      AllowlistPath: docsms-allowlist/python-allowlist.txt
+    - RepoName: azure-docs-sdk-node
+      RepoOwner: MicrosoftDocs
+      AllowlistPath: docsms-allowlist/js-allowlist.txt
+
+- name: TargetBranch
+  type: string
+  default: live
+
+jobs:
+- ${{ each repo in parameters.DocsRepos }}:
+  - job: 
+    displayName: 'ReleaseDocsMs: ${{repo.RepoName}}'
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+      vmImage: MMSUbuntu20.04
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+    steps:
+      # Checkout tool repo
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+      # Checkout docs repo
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          SkipDefaultCheckout: true
```

Comment:
FYI @benbp I know you are transitioning away from this name so once this goes in we will need to update it as part of your removal of this name.

Reply:
The change is merged. I will update mine accordingly